ui/package.json

```diff
@@ -10,8 +10,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "oidc-client-ts": "^3.0.1",
```

Comment:
If there was a choice between different provider, please explain in the commit message why you chose this one.

Reply:
No real alternatives that I know of.